### ISSUE TITLE
RCS1055 for empty enums

### ISSUE THREAD

**Issue by jnyrup** [dotnet/roslynator#1359]
**Product and Version Used**: 
Using newest v4.9.0 RCS1055 is triggered on enums.
It does not trigger on class, struct or interface, which looks related to #1134.

**Steps to Reproduce**:
```
root = true

[*.cs]
dotnet_diagnostic.RCS1055.severity = error
```

```cs
class Class;

enum Enum; // <-- RCS1055

interface IInterface;

struct Struct;
```

**Actual Behavior**:
RCS1055 is reported.

**Expected Behavior**:
RCS1055 should not be reported.
